selenium-testing/src/test/java/ca/bc/gov/ecc/ofm/selenium/v1/tests/PortalCreateAllowanceApplication.java

```diff
@@ -0,0 +1,97 @@
+package ca.bc.gov.ecc.ofm.selenium.v1.tests;
+
+import java.time.Duration;
+import org.openqa.selenium.*;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.annotations.AfterTest;
+import org.testng.annotations.BeforeTest;
+import org.testng.annotations.Test;
+
+import com.aventstack.extentreports.MediaEntityBuilder;
+
+import ca.bc.gov.ecc.ofm.selenium.v1.PageFactory_Portal.*;
+import io.github.bonigarcia.wdm.WebDriverManager;
+
+public class PortalCreateAllowanceApplication extends BaseTest {
+
+	WebDriver driver;
+
+	@BeforeTest
+	public void initDriver() {
+		WebDriverManager.chromedriver().setup();
+		driver = new ChromeDriver();
+		driver.manage().window().maximize();
+	}
+
+	public void wait(int t) {
+		driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(t));
+	}
+
+	@SuppressWarnings("deprecation")
+	@Test
+	public void CreateAllowanceApplication() throws Exception {
+		try {
+			driver.get(PORTAL_URL);
+		//	test = extent.createTest("Test - Portal Create Application");
```

Comment:
Please uncomment. It doesn't run for me when this is commented

Reply:
uncommented